lib/fxpay/index.js

```diff
@@ -56,13 +56,20 @@
 
 
   exports.purchase = function _purchase(productId, onPurchase, opt) {
-    opt = opt || {};
-    opt.maxTries = opt.maxTries || undefined;
-    opt.pollIntervalMs = opt.pollIntervalMs || undefined;
+    opt = utils.defaults(opt, {
```

Comment:
Ah ok, fair enough then. Maybe a comment with the reasoning - just for our future selves?